content/post/My-journey-with-ALiAS.md

```diff
@@ -0,0 +1,37 @@
++++
+title = "My journey with ALiAS : From a commit to the community"
+slug = "FOSS communities"
```

Comment:
Slugs don't work well with capitals and spaces. How about something unique like `animeshaliasjourney`

Reply:
> Slugs don't work well with capitals and spaces. How about something unique like `animeshaliasjourney`

I guess `alias journey` would be simpler and better.

Reply:
A slug should be self-explanatory if a person just reads it from the URL. 
Since there will be a lot of journey posts hence it's better to name them 

Better Slud - animesh-alias-journey

Reply:
> Better Slug - `animesh-alias-journey`

LGTM